src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -434,7 +434,7 @@ archiveBuildUpload apiOpts archive = runEmpty $
     let opts = "dependency" =: True <> "rawLicenseScan" =: True
     -- The API route expects an array of archives, but doesn't properly handle multiple archives so we upload
     -- an array of a single archive.
-    let archiveProjects = ArchiveComponents [archive]
+    let archiveProjects = ArchiveComponents [archive] False
```

Comment:
We'll make this respect the `--force-vendored-dependency-rescans` flag when we do [ANE-361](https://fossa.atlassian.net/browse/ANE-361)